### ISSUE TITLE
Installer cannot mount cdrom with joliet info

### ISSUE THREAD

**Issue by Raqbit** [rancher/k3os#395]
**Version (k3OS / kernel)**
Commits after 794ce9f14aa6845d084c5d6e06da224c91fdf37e

**Architecture**
x86_64

**Describe the bug**
The k3OS installer fails to mount the cdrom (ISO file) and drops to a rescue shell.

**To Reproduce**
Run the following with a k3OS ISO, kernel and initrd after commit 794ce9f14aa6845d084c5d6e06da224c91fdf37e. (And compare with ISO's before that commit, like `0.9.0`)

config.yaml:
```json
ssh_authorized_keys:
- github:raqbit

k3os:
  k3s_args:
  - server
  - "--no-deploy"
  - "traefik"
```

start.sh:
```bash
kernel_args="\
console=ttyS0 \
k3os.mode=install \
k3os.install.silent=true \
init_cmd=\"cp /.base/config.yaml /config.yaml\"
k3os.install.config_url=/config.yaml \
k3os.install.device=/dev/vda \
k3os.hostname=k3os \
k3os.install.debug=true \
k3os.debug=true \
"

virt-install \
	--connect qemu:///session \
	--name k3os \
	--disk size=8 \
	--ram 2048 \
	--vcpus 1 \
	--os-type linux \
	--os-variant ubuntu18.04 \
	--graphics none \
	--network bridge=virbr0 \
	--disk path=k3os-amd64.iso \
	--install kernel=k3os-vmlinuz-amd64,initrd=k3os-initrd-amd64 \
	--initrd-inject config.yaml \
	--extra-args "${kernel_args}"
```

**Expected behavior**
It should mount the ISO like it did on 0.9.1

```
++ source /usr/libexec/k3os/live
+++ setup_base
++++ blkid -L K3OS
++++ true
+++ K3OS_ISO=
+++ '[' -n '' ']'
+++ success=false
+++ (( j=0 ))
+++ (( j < 5  ))
++++ grep -w disk
++++ awk '{print $1}'
++++ lsblk -o NAME,TYPE -n
+++ for i in $(lsblk -o NAME,TYPE -n | grep -w disk | awk '{print $1}')
+++ mount -t iso9660 /dev/vda /.base
[    2.881385] ISOFS: Unable to identify CD-ROM format.
mount: /.base: wrong fs type, bad option, bad superblock on /dev/vda, missing codepage or helper program, or other error.
+++ for i in $(lsblk -o NAME,TYPE -n | grep -w disk | awk '{print $1}')
+++ mount -t iso9660 /dev/vdb /.base
mount: /.base: WARNING: device write-protected, mounted read-only.
+++ success=true
+++ break
+++ '[' true = true ']'
+++ break
+++ setup_kernel
```

**Actual behavior**
With `k3os.debug` and `k3os.install.debug` enabled, the following output is given before the k3OS installer drops to a rescue shell: 

```
++ source /usr/libexec/k3os/live
+++ setup_base
++++ blkid -L K3OS
+++ K3OS_ISO=/dev/vdb3
+++ '[' -n /dev/vdb3 ']'
++++ blkid -L K3OS
+++ mount -t iso9660 -o ro /dev/vdb3 /.base
[    2.874922] ISOFS: Unable to identify CD-ROM format.
mount: /.base: wrong fs type, bad option, bad superblock on /dev/vdb3, missing codepage or helper program, or other error.
+ rescue
+ echo ERROR 'Something went wrong, run with cmdline k3os.debug for more logging'
ERROR Something went wrong, run with cmdline k3os.debug for more logging
+ echo Dropping to shell
Dropping to shell
+ exec bash
bash: cannot set terminal process group (-1): Not a tty
bash: no job control in this shell
bash-5.0# [    2.946300] pciehp 0000:00:01.6:pcie004: Failed to check link status
```

**Additional context**

Versions:
```
qemu: 4.0.0
virt-install: 2.2.1
```

**Comment by dweomer** [comment]
`/dev/vdb3` ?

**Comment by dweomer** [comment]
@Raqbit I think I have this fixed in [v0.10.0-dev1](https://github.com/rancher/k3os/releases/tag/v0.10.0-dev1).

I requisitioned a NUC and installed Ubuntu Eaon then installed virt-manager and qemu (same versions as you sepcified). Using your script with the v0.10.0-dev1 ISO booted without problem.

**Comment by dweomer** [comment]
Okay, fixed again via #429
